.github/workflows/blobstorage-preview.yml

```diff
@@ -0,0 +1,33 @@
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID_BLOBS }}
```

Comment:
better to follow the same naming convention. i think that would be `VERCEL_PROJECT_ID_BLOBSTORAGE`